sound/soc/sof/core.c

```diff
@@ -19,7 +19,10 @@
 #include <trace/events/sof.h>
 
 /* see SOF_DBG_ flags */
-static int sof_core_debug =  IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE);
+static int sof_core_debug =  IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_FIRMWARE_TRACE) |
+				SOF_DBG_SQUELCH_DMA_POSITION_UPDATE_LOGS |
+				SOF_DBG_SQUELCH_IPC_SUCCESS_LOGS;
```

Comment:
It might be better to have opt-in type of flags? SOF_DBG_PRINT_*
From one of the logs in the CI run:
```
[  217.953676] kernel: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx: 0x30200000: GLB_TPLG_MSG: BUFFER_NEW
[  217.953819] kernel: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG: DMA_POSITION
[  217.953845] kernel: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG: DMA_POSITION
[  217.953858] kernel: sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx succeeded: 0x30200000: GLB_TPLG_MSG: BUFFER_NEW
```
https://sof-ci.01.org/linuxpr/PR3631/build100/devicetest/?model=CML_SKU0955_HDA&testcase=check-playback-10sec

The trace is explicitly enabled by `sof_debug` module parameter in CI, so it disables the silencing of the prints.

A compromise might be to still show the dtrace position updates, but tap into the SOF_DBG_SQUELCH_IPC_SUCCESS_LOGS and omit the `ipc rx done` along the `ipc tx succeeded`, so we would still have information that an IPC RX happened, but no double printing.
Fwiw, the !CONFIG_SND_SOC_SOF_DEBUG_VERBOSE_IPC case skips anything SOF_IPC_GLB_TRACE_MSG.

Reply:
I can't follow your comment @ujfalusi.

Reply:
@plbossart, what I meant is:
s/SOF_DBG_SQUELCH_IPC_SUCCESS_LOGS/SOF_DBG_PRINT_IPC_SUCCESS_LOGS
s/SOF_DBG_SQUELCH_DMA_POSITION_UPDATE_LOGS/SOF_DBG_PRINT_DMA_POSITION_UPDATE_LOGS

If SOF_DBG_PRINT_IPC_SUCCESS_LOGS is _not_ set, we will not print success for tx/rx
if SOF_DBG_PRINT_DMA_POSITION_UPDATE_LOGS is _not_ set, we will hide the position updates for trace.

In case someone wants to see them, the flags must be explicitly set. This will make sure that we are reducing the noise by default.

Reply:
ok, will change that, thanks for the clarification.

Reply:
done